commands/make/make.drush.inc

```diff
@@ -760,8 +795,8 @@ function drush_make_convert_version_to_composer($version) {
       $cver = $version;
     }
     else {
-      // Replace '1.x' with '^1.*'.
-      $cver = '^' . str_replace('x', '*', $version);
+      // Replace '1.x' with '1.*'.
+      $cver = str_replace('x', '*', $version);
```

Comment:
I believe this is like https://github.com/drush-ops/drush/pull/4082/files#diff-cca65dcdb439384668dcd83746546302R652